roles/deploy/tasks/deploy.yml

```
@@ -114,14 +114,9 @@
   changed_when: False
   tags: rake
 
-- name: load schema if it isn't loaded
-  command: bash -lc "bundle exec rake db:schema:load RAILS_ENV={{ rails_env }} I_AM_SURE=1"
-  args:
-    chdir: "{{ build_path }}"
+- name: prepare database
```

When reading the file it's not immediately clear this task is conditional, so I'd add a clue in the task name (like there used to be). My suggestion:

```suggestion
- name: prepare database for the first time
```